common/djangoapps/student/admin.py

```diff
@@ -217,8 +219,17 @@ def has_module_permission(self, request):
         return self.has_permission(request, 'has_module_permission')
 
 
+class NameEnForm(forms.ModelForm):
+    name_en = forms.CharField(required=False, widget=widgets.TextInput)
+
+    def save(self, commit=True):
+        self.instance.name_en = self.cleaned_data['name_en']
+        super(NameEnForm, self).save()
```

Comment:
You are not passing `save` method args and kwargs to the super `save`. I would recommend:
```python
def save(self, *args, **kwargs):
    ...
    super(NameEnForm, self).save(*args, **kwargs)
```